src/old-components/Guilds/ActionsBuilder/SupportedActions/SetPermissions/AddressInput.tsx

```diff
@@ -0,0 +1,62 @@
+import { ParsedDataInterface, ValidationsInterface } from './types';
+import { ClickableIcon } from './styles';
+import Input from 'old-components/Guilds/common/Form/Input';
+import Avatar from 'old-components/Guilds/Avatar';
+import { FiX } from 'react-icons/fi';
+
+interface AddressButtonProps {
+  customToAddress: string;
+  anyAddressToggled: boolean;
+  validations: ValidationsInterface;
+  destinationAvatarUrl: any;
+  parsedData: ParsedDataInterface;
+  handleCustomAddress: (value: string) => void;
+}
+
+const AddressInput: React.FC<AddressButtonProps> = ({
+  customToAddress,
+  anyAddressToggled,
+  validations,
+  destinationAvatarUrl,
+  parsedData,
+  handleCustomAddress,
+}) => {
+  return (
+    <Input

```

Comment:
Would it be possible for us to reuse the existing address component here and just add on the "any" functionality?

Reply:
I don't seem to find an address input component, or maybe I'm not understanding the question

Reply:
/src/old-components/Guilds/common/Form/AddressInput.tsx

Reply:
Done! It was a perfect fit.

I also added a "clear input" button to the component.